### DISCUSSION TITLE
Feat ⭐️: Add scroll to top button

### DISCUSSION THREAD

**Discussion by ArslanYM** [ArslanYM/StarterHive#208] [Ideas]
Lets talk @

**Comment by yacdanyCR** [comment]
@ArslanYM  Ther's a progress on this ? if not I could work on this feature , I have in mind something like the pictures below.

### **Button Behavior**

1. The button dissapear when the User is on the top of the  webpage.
2. The button appear when the User Scroll Down. 
3. When the User Click on the Button , the scroll will direct the user to the top of the page and the button disappears.


![Screenshot (71) - Copy](https://github.com/ArslanYM/StarterHive/assets/77762960/4404d5f9-9081-40e7-b663-7281139a5b13)
![Screenshot (71)](https://github.com/ArslanYM/StarterHive/assets/77762960/6da2525e-1f05-47a0-8cbb-669dbe29c1d6)

**Reply by ArslanYM** [reply]
@yacdanyCR , I dont think we will need this feature for all pages. We will only need this in the issues section , where issues will be listed. Also  can we iterate on the UI of the scroll to top button.

**Reply by yacdanyCR** [reply]
What do you mean with “iterate” ? Like a pagination in the UI of the scroll button ?

**Reply by ArslanYM** [reply]
By iterate I mean make the button look better. Since this feature isnt necessary, but we are adding it anyways. It should be atleast better looking YKWIM ?

**Reply by yacdanyCR** [reply]
If in the future if  you have 20 or more cards you don't want to be scrolling down and the scroll button will be less necessary, the best thing for the issue page is create a  pagination  and discard the scroll button but if you want the scroll button I can work on that

**Reply by ArslanYM** [reply]
Ive no clue about a pagination. Please enlighten me

**Reply by yacdanyCR** [reply]
Something like this https://flowbite.com/docs/components/pagination/ so you can show 5 cards or 8 per page and the User don’t need to scroll down and down

**Reply by ArslanYM** [reply]
> Something like this https://flowbite.com/docs/components/pagination/ so you can show 5 cards or 8 per page and the User don’t need to scroll down and down

This seems like a cool idea, do you think we should incorporate this now or in the future?

**Reply by WajeehaCodes** [reply]
Sometimes, scroll button is little handy and works fine but pagination is a better idea. Scroll button can be helpful in some other pages

**Reply by ArslanYM** [reply]
Okay then , lets add a pagination. Looks minimalist as well.

**Reply by ThGnommy** [reply]
Maybe a 'scroll to top button' could be helpful when the user is on mobile/tablet? I thought about the docs and the contributor pages. Especially the contributor page, the more the number of contributors grows more the page "grows" vertically. I can help with this feature.

**Reply by ArslanYM** [reply]
@ThGnommy Yeah, it seems that there has been no progress on this from @yacdanyCR . @ThGnommy You can start working on it. The issue number is : #199

**Reply by ThGnommy** [reply]
Thank you!

**Comment by ArslanYM** [comment]
@WajeehaCodes @ThGnommy Have you joined the discord server?

**Reply by ThGnommy** [reply]
Not yet! Thanks for remembering me. I'll join it asap.

**Reply by WajeehaCodes** [reply]
Can I get the link?

**Reply by ArslanYM** [reply]
Sure , https://discord.com/invite/VkBTKUeVnG

**Reply by WajeehaCodes** [reply]
Thanks
